frontend/src/views/sql-editor/ConnectionPanel/ConnectionPane/TreeNode/DatabaseNode.vue

```diff
@@ -24,20 +24,28 @@
     <span class="flex-1 truncate">
       <HighlightLabelText :text="database.databaseName" :keyword="keyword" />
       <span v-if="!hasInstanceContext" class="text-control-light">
-        ({{ database.instanceEntity.title }})</span
-      >
+        ({{ database.instanceEntity.title }})
+      </span>
     </span>
+    <RequestQueryButton
```

Comment:
Maybe it should be hidden in standalone mode.

Reply:
Updated